static/assets/js/core.js

```diff
@@ -104,4 +104,18 @@ $(document).ready(function() {
       });
     });
   }
+
+  /**Dark mode controller */
+  var darkmode = localStorage.getItem('dark-mode');
```

Comment:
`localStorage` doesn't work in private mode in some browsers (I learnt it the hard way :) ). I think we can use `local-storage-polyfill.js` here which falls back to `cookie` automatically, here's how it's currently used in the code https://github.com/simple-login/app/blob/70ce48cd792b85914147c96bc70babbc4083edac/server.py#L475

Reply:
I saw the local storage polyfill but didn't know that it has problems with private mode. I'm going to update it